src/index.js

```diff
@@ -1,17 +1,115 @@
 import './style.css';
-import {
-  render,
-  displayCheckbox,
-  checkCheckbox,
-} from './checkbox.js';
-
-window.addEventListener('load', () => {
-  render();
-  displayCheckbox();
-  const checkbox = document.querySelectorAll('.checkbox');
-  for (let i = 0; i < checkbox.length; i += 1) {
-    checkbox[i].addEventListener('change', () => {
-      checkCheckbox(i);
+import Checkbox from './checkbox.js';
+import Storage from './inputRecieve.js';
+import Tasks from './task.js';
+
+const tasksDisplay = document.querySelector('.tasks');
+
+class Ui {
+  static displayTasks() {
+    const tasks = Storage.getStorage();
+
+    tasks.forEach((task) => {
+      this.render(task);
+    });
+  }
+
+  static render(task) {
+    tasksDisplay.innerHTML += `
+    <li class="displayList">
+      <div class="innerItems">
+        <input class="checkbox" type="checkbox">
+        <p class="inputDisplay" contentEditable = true>${task.description}</p>
+      </div>
+      <div class="innerItems">
+        <button class="listButton"><i class='fas fa-ellipsis-v'></i></button>
+        <button class="binButton"><i class="material-icons" id="bin">delete</i></button>
+      </div>
+    </li>
+    `;
+  }
+
+  static clearList() {
+    tasksDisplay.innerHTML = '';
+  }
+
+  static clearInput() {
+    document.querySelector('.input').value = '';
+  }
+
+  static displayDeleteButton(el) {
+    if (el.classList.contains('listButton')) {
+      el.classList.toggle('buttonDisappear');
+      el.nextElementSibling.classList.toggle('buttonAppear');
+    }
+  }
```

Comment:
- [ ] This method doesn't seem working because when I click the `listButton` icon it is non-responsive. I should be able to delete an item without the `clear all completed` button, so that this method should display the `binButton` when the `listButton` is clicked. Kindly fix this issue. :pray:

Reply:
Hello, Henok. You are reviewing my project. To access the delete button you can click on the display list words (editable words). I moved it because it is similar to the example we were shown.
Similar to this example. https://web.archive.org/web/20180320194056/http://www.getminimalist.com:80/
The list icon is used to move the items which was the optional function.